salt/journal-cms/init.sls

```diff
@@ -384,6 +384,18 @@ syslog-ng-for-journal-cms-logs:
         - listen_in:
             - service: syslog-ng
 
+
+{% if pillar.elife.env == 'end2end' %}
+populate-people-api-with-fixtures:
+    cmd.script:
+        - name: ../vendor/bin/drush create-person reviewing-editor "Higgins" --given="Henry" --email="henry.higgins@myfairlady.co.uk" --match
```

Comment:
Currently, if `--match` is used and no match is found then it will not create the user. If this is needed for idempotence then I will adapt the script.

Reply:
Right, it's easier to always run the same command in an upsert scenario rather than doing more `drush` calls to find out if the user is there

Reply:
I will prepare a PR to Journal CMS to allow for this.

Reply:
Once we have merged the following PR we can use the `--upsert` flag: https://github.com/elifesciences/journal-cms/pull/366